exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/decoder/Packet.java

```diff
@@ -52,6 +52,7 @@
   private int packetLength;
   protected int etherProtocol;
   protected int protocol;
+  protected boolean isCorrupt = false;
```

Comment:
This was a brilliant idea! Ted's comment on the mailing list was spot-on and even just knowing which packet was "interesting" (in a bad/corrupt way) is super-helpful for identifying potentially malicious behavior.